sklearn/svm/src/newrand/newrand.h

```diff
@@ -5,12 +5,12 @@
      libsvm and liblinear.
      Sylvain Marie, Schneider Electric
      See <https://github.com/scikit-learn/scikit-learn/pull/13511#issuecomment-481729756>
-
  */
 #ifndef _NEWRAND_H
 #define _NEWRAND_H
 
 #ifdef __cplusplus
+#include <random>
```

Comment:
Any reason for adding this  `include` ? It did not seem to be useful before

Reply:
That's so cython can create a .cpp file from newrand.h

Reply:
ok I see. Maybe we can add this as a comment before the include line ?